app/views/admin/applications/show.html.erb

```diff
@@ -0,0 +1,23 @@
+<h3>Applicant Details</h3>
+<p><strong>Applicant Name:</strong> <%= @application.applicant_name %></p>
+<p><strong>Address:</strong> <%= @application.address %></p>
+<p><strong>City:</strong> <%= @application.city %></p>
+<p><strong>State:</strong> <%= @application.state %></p>
+<p><strong>Zip code:</strong> <%= @application.zip_code %></p>
+<p><strong>Why I would make a good pet owner:</strong> <%= @application.description %></p>
+
+<p><strong>Applying for pets:</strong></p>
+
+<% @application.pet_applications.each do |pet_application| %>
+  <div id="pet_<%= pet_application.pet.id %>">
+    <p><%= pet_application.pet.name %></p>
+    <% if pet_application.status.nil? %>
+      <%= button_to "Approve", "/admin/applications/#{@application.id}/pets/#{pet_application.pet.id}/approve", method: :patch %>
```

Comment:
Instead of having 2 different paths and 2 different controller actions for this, consider adding parameters to this button_to by adding `params: { approved: true }` or false if rejecting. Then this would just direct to an `update` controller action that could handle both cases by looking at `params[:approved]`